.github/ISSUE_TEMPLATE.md

```diff
@@ -30,6 +30,9 @@ What behavior did you expect to observe?
 **Execution command:**
 <!-- e.g. `./foo 42`. If an input file is required, include it as well. -->
 
+**Associated Future Test(s):**
+<!-- Are there any tests in Chapel's test system that demonstrate this issue?
+     e.g. test/path/to/foo.future #1234 -->
```

Comment:
I agree about the tradeoff here.. I wonder if we might put Compile Command and Execution Command first, followed by Source Code and Associated Future Tests. (if we're not too attached to the current order).